apps/web/src/components/views/ProfileView/components/ProfileEmailPrimary.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react"
+import * as DropdownMenu from "@radix-ui/react-dropdown-menu"
```

Comment:
This should be a sepperate component imo and probably a part of the UI package

Reply:
Email tab is removed anyways, removed the files now :)